redisinsight/ui/src/pages/clusterDetails/components/cluser-nodes-table/ClusterNodesTable.tsx

```diff
@@ -0,0 +1,195 @@
+import {
+  EuiBasicTableColumn,
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiLoadingContent,
+  EuiToolTip,
+  PropertySort
+} from '@elastic/eui'
+import { IconType } from '@elastic/eui/src/components/icon/icon'
+import cx from 'classnames'
+import { map } from 'lodash'
+import React, { useState } from 'react'
+import {
+  InputIconSvg,
+  KeyIconSvg,
+  MemoryIconSvg,
+  OutputIconSvg,
+  UserIconSvg,
+  MeasureIconSvg
+} from 'uiSrc/components/database-overview/components/icons'
+import { ModifiedClusterNodes } from 'uiSrc/pages/clusterDetails/ClusterDetailsPage'
+import { formatBytes, Nullable } from 'uiSrc/utils'
+import { numberWithSpaces } from 'uiSrc/utils/numbers'
+
+import styles from './styles.module.scss'
+
+const ClusterNodesTable = ({ nodes, loading }: { nodes: Nullable<ModifiedClusterNodes[]>, loading: boolean }) => {
+  const [sort, setSort] = useState<PropertySort>({ field: 'host', direction: 'asc' })
+
+  const isMaxValue = (field: string, value: number) => {
```

Comment:
Maybe will be better move it to utils or helper

Reply:
I'm not sure, very specific function.